docs/integrations/keras_integration.rst

```diff
@@ -0,0 +1,597 @@
+
+
+
+Step 2: In your notebook (colab/local) and install neptune client library.
+
+.. code-block:: bash
+
+   pip install neptune-contrib
+
+
+STEP 3: Import neptune, and add your initialization code:
+
+.. code-block:: python3 
+
+   import neptune
+   neptune.init(api_token='ADD-API-TOKEN-HERE',project_qualified_name='risingodegua/spam-classifier')
+
+The code above will initialize your project, and connect your local neptune client to your online account. So that any logs made locally can be synced online.
+
+**Note:** Your API token is a secret, and should be hidden from outsiders. In production environments, it is advisable to add the token to your environment variable. 
+Neptune will automatically pick up the variable from the environment as long as it is called NEPTUNE_API_TOKEN, so you can easily initialize Neptune by running:
+
+.. code-block:: python3
+
+   neptune.init(project_qualified_name='risingodegua/spam-classifier')
+
+
+
+STEP 4: Create your first experiment under your project
+
+.. code-block:: bash
+
+   neptune.create_experiment(name='spam_exploration')
+
+.. image:: ../_static/images/integrations/keras-integration/out1.png
+
+
+Experiments are important for tracking and experimenting on different ideas, and aids reproduction. It is important to always create a new experiment for any new idea you want to try out.
+The |experiment link| provided will open the experiment dashboard where you can see details about your experiments in real time. This link can be shared with collaborators and even the general public if the project has been made public.
+Note: Neptune also tracks hardware metrics to tell you how your machine operates during experimentation. If you're running this experiment locally, you may have to install psutil library first.
+
+.. |experiment link| raw:: html
+
+    <a href="https://ui.neptune.ai/risingodegua/spam-classifier/e/SPAM-2" target="_blank">experiment link</a>
+
+.. code-block:: bash
+
+   pip install psutil
+
+To see your hardware metrics, you can click on the project |link| generated by Neptune above, and then select logs. You should be presented with a realtime dashboard as shown below:
+
+.. image:: ../_static/images/integrations/keras-integration/kix.j8hadw1vnql1.png
+
+
+
+Step 5: In a new cell of your notebook, import all the necessary packages needed for this project.
+
+.. code-block:: python3
+
+   import numpy as np
+   import pandas as pd
+   import matplotlib.pyplot as plt
+   import seaborn as sns
+   import wordcloud
+   import warnings
+   warnings.filterwarnings("ignore")
+
+   from neptunecontrib.monitoring.keras import NeptuneMonitor
+
+Notice that we're importing the NeptuneMonitor from the neptunecontrib package? This is because we are going to be integrating neptune with keras. Neptune can also be integrated with other libraries easily. You can find some useful link on how to achieve this below.
+
+**TODO: SOME SEO LINKS HERE**
+
+
+IMPORT DATA AND PERFORM SOME SIMPLE EDA
```

Comment:
This should definitely not be a part of the documentation page. 

I think this is very valuable for a blog post -> perhaps we should convert what you wrote here to a blog post as well.